docs/guides/embedding-content.rst

```diff
@@ -0,0 +1,50 @@
+Embedding content from your docs online
+=======================================
+
+Using the `Read the Docs JavaScript Client`_, or with basic calls to our REST
+API, you can retrieve embeddable content for use on your own site. Content is
+embedded in an iframe element, primarily for isolation. 
```

Comment:
> Content is embedded in an iframe element, primarily for isolation.

I'm not sure about this phrase. sphinx-hoverxref doesn't work like that --it just add the content retrieved directly into the DOM.